orchestrators/ProcurementOperations/tests/Unit/Coordinators/GoodsReceiptCoordinatorTest.php

```
@@ -0,0 +1,97 @@
+            public function postGoodsReceiptAccrual(
+                string $tenantId,
+                string $goodsReceiptId,
+                string $purchaseOrderId,
+                array $lineItems,
+                string $receivedBy
+            ): string {
+                return 'accrual-1';
+            }
+        };
+
+        $goodsReceiptPersistStub = new class {
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $goodsReceiptPersistStub. Keep variable name length under 20. (undefined)

(LongVariable)